.storybook/pages/CoursePlannerEdit/CoursePlannerEdit.tsx

```diff
@@ -1,6 +1,8 @@
 import React, { useState } from 'react';
 import styles from './CoursePlannerEdit.module.css';
 
+import ProjectCard from './ProjectCard';
```

Comment:
Not blocking, but if it's solely gonna be used in here, and not have stories for `ProjectCard`, the code could just live in here